src/Uno.UI/WasmCSS/Uno.UI.css

```diff
@@ -174,6 +174,7 @@ embed.uno-frameworkelement.uno-unarranged {
   user-select: none;
   -webkit-user-drag: none;
   user-drag: none;
+  pointer-events: none;
```

Comment:
This is already applied to `.uno-uielement`:

https://github.com/unoplatform/uno/blob/337801acc17433fb31195ed925ef922a6c16a376/src/Uno.UI/WasmCSS/Uno.UI.css#L28-L35

Shouldn't HtmlImage already be a UIElement and already have `pointer-events: none`? Am I missing anything?